web-app/src/handlebars/helpers.ts

```diff
@@ -71,6 +71,22 @@ export function registerHandlebarsHelpers() {
       );
     },
   );
+  handlebars.registerHelper(
+    'isCellSuggested',
+    function (
+      suggestedCell: Coordinates,
+      cellXCoord: number,
+      cellYCoord: number,
+      turn: StoneColor,
```

Comment:
Shouldn't the server send an hint only when it's the player's turn ? I really don't have an answer to this so it's ok for me if we don't touch it for now.